<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0 (Python 3.11.3 on win32)" generated="2024-02-13T19:36:08.185371" rpa="false" schemaversion="5">
<suite id="s1" name="basicScript" source="E:\QA Sanbercode\belajar-robot-sb53\day-1\basicScript.robot">
<test id="s1-t1" name="Login with Invalid Data" line="38">
<kw name="Open Flight Application">
<kw name="Open Application" owner="AppiumLibrary">
<kw name="Capture Page Screenshot" owner="AppiumLibrary">
<msg time="2024-02-13T19:36:13.169251" level="FAIL">No application is open</msg>
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<status status="FAIL" start="2024-02-13T19:36:13.169251" elapsed="0.251552">No application is open</status>
</kw>
<msg time="2024-02-13T19:36:13.425789" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg time="2024-02-13T19:36:13.427792" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
    at getResponseForW3CError (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)</msg>
<arg>remote_url=${REMOTE_URL}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=12.0</arg>
<arg>deviceName=emulator-5554</arg>
<arg>appPackage=com.example.myapplication</arg>
<arg>appActivity=com.example.myapplication.MainActivity</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<status status="FAIL" start="2024-02-13T19:36:09.654967" elapsed="3.776825">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
    at getResponseForW3CError (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)</status>
</kw>
<status status="FAIL" start="2024-02-13T19:36:09.651970" elapsed="3.779822">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
    at getResponseForW3CError (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)</status>
</kw>
<kw name="Click Sign In Button On Home Page">
<status status="NOT RUN" start="2024-02-13T19:36:13.435792" elapsed="0.000000"/>
</kw>
<kw name="Input Username">
<status status="NOT RUN" start="2024-02-13T19:36:13.436792" elapsed="0.000000"/>
</kw>
<kw name="Input Password">
<status status="NOT RUN" start="2024-02-13T19:36:13.438791" elapsed="0.000000"/>
</kw>
<kw name="Click Sign In Button On Login Page">
<status status="NOT RUN" start="2024-02-13T19:36:13.441793" elapsed="0.000000"/>
</kw>
<kw name="Verify User Is Fail To Logged In">
<status status="NOT RUN" start="2024-02-13T19:36:13.442794" elapsed="0.000000"/>
</kw>
<kw name="Close Flight Application">
<status status="NOT RUN" start="2024-02-13T19:36:13.463789" elapsed="0.000000"/>
</kw>
<status status="FAIL" start="2024-02-13T19:36:09.570972" elapsed="3.895818">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 12.0. The following are available: 421d411b (14)
    at getResponseForW3CError (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Administrator\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)</status>
</test>
<status status="FAIL" start="2024-02-13T19:36:08.191376" elapsed="5.295411"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="basicScript">basicScript</stat>
</suite>
</statistics>
<errors>
<msg time="2024-02-13T19:36:13.425789" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
